### ISSUE TITLE
Problem on layers concatenation

### ISSUE THREAD

**Issue by alexbarnadas** [ambekarsameer96/GLSS#2]
Hello, I'm having this trouble and as I am no expert and I still don't really understand what each hyperparameters does I was wondering if there was a quick solution to this problem:
`A 'Concatenate' layer requires inputs with matching shapes except for the concat axis. Got inputs shapes: [(None, 16, 16, 64), (None, 1, 1, 8)]`

The only thing I had to change (in the layer part of the code) was, to import `from efficientnet.keras import EfficientNetB4` instead of `from efficientnet import EfficientNetB4`

Do you (or does anyone who reads this) have a clue of what shoud I touch?

**Comment by Aayushktyagi** [comment]
Hey @alexbarnadas library efficientnet (https://github.com/qubvel/efficientnet) got updated. These shape mismatches are due to that. 
You can try pip install -U efficientnet==0.0.4
Thanks

**Comment by alexbarnadas** [comment]
Thanks a lot it solved my problem!
